public/index.html

```diff
@@ -4,35 +4,42 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Express Forum</title>
     <link rel="stylesheet" href="style.css" />
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.0/jquery.min.js"></script>
   </head>
   <body>
     <div class="container">
-      <div class="top-bar">
-        <h1>Question Forum</h1>
-      </div>
-      <br />
-      <br />
-      <div class="threads">
-        <header>Thread 1</header>
-      </div>
-      <div class="row">
-        <li class="comment-content">Comments</li>
-      </div>
-
-      <textarea>text area</textarea>
-      <br />
-      <button type="button" id="add">Add comment</button>
+      <h1>Threads</h1>
+      <div id="thread-display"></div>
+    </div>
 
-      <div class="threads"><header>Thread 2</header></div>
-      <div class="row">
-        <li class="comment-content">Comment here</li>
-      </div>
-      <textarea>text area</textarea>
+    <script type="text/html" id="thread-template">
+      <h2>${title}</h2>
+      <cite>- ${author}</cite>
+      <h3>Comments:</h3>
+      <ul>
+        {{each comments}}
```

Comment:
`{{ each comments }}` is an iterator that iterates over the comments property of the data that is passed to this template. So, for each  element in `comments`, we'll display an `<li>` with `${content}` and `${author}`.

The jQuery template library is really handy for converting JSON that comes back in an API response into HTML to render on your web page.